packages/next/bundles/webpack/bundle5.js

```diff
@@ -0,0 +1,12 @@
+/* eslint-disable import/no-extraneous-dependencies */
+
+module.exports = function () {
+  return {
+    BasicEvaluatedExpression: require('webpack5/lib/javascript/BasicEvaluatedExpression'),
+    ModuleFilenameHelpers: require('webpack5/lib/ModuleFilenameHelpers'),
+    NodeTargetPlugin: require('webpack5/lib/node/NodeTargetPlugin'),
+    StringXor: require('webpack5/lib/util/StringXor'),
+    sources: require('webpack-sources'),
```

Comment:
Should this be `require('webpack5').sources`?

Reply:
Yes, good catch.